cli/src/cluster_query.rs

```diff
@@ -430,7 +430,7 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .index(1)
                         .value_name("ADDRESS")
                         .required(true),
-                    "Account address"
+                    "Account."
```

Comment:
nit: This one seems a little goofy, it would expand to:
```
Account. Address is one of:
```
I think we could either add more information (my suggested text here similar to address text you modified in `cli/src/inflation.rs`:
```
Account to query for transactions.
```
or maybe even go empty string here so it is just:
```
Address is one of:
```
after the macro expands